### PR TITLE
TVR-21905: Request Allocations via API v4 - Swagger page

### PR DESCRIPTION
**Pull request by ArjitGarg-SAP**

### CODE REVIEW

File: src/api-explorer/v4-0/ConcurRequest.swagger2.json [hunk 1]
```diff
@@ -2505,6 +2509,189 @@
...
+                "REQ_PROCESSOR1",
+                "REQ_PROCESSOR2",
+                "REQ_PROCESSOR3",
+                "RISK_PROCESSOR"
+            ]
+          },
+          {
+            "name": "userId",
+            "in": "query",
+            "description": "The unique identifier of the userto update the allocation of a Request",
```
Comment (quancheng-zhao) [inline src/api-explorer/v4-0/ConcurRequest.swagger2.json:2636]: The unique identifier of the user to update the allocation of a Request

File: src/api-explorer/v4-0/ConcurRequest.swagger2.json [hunk 2]
```diff
@@ -2505,6 +2509,189 @@
...
+            "in": "path",
+            "description": "The unique identifier of the allocation",
+            "required": true,
+            "type": "string"
+
+          },
+          {
+            "name": "userId",
+            "in": "query",
+            "description": "The unique identifier of the userviewing the allocation of a Request",
```
Comment (quancheng-zhao) [inline src/api-explorer/v4-0/ConcurRequest.swagger2.json:2546]: The unique identifier of the user viewing the allocation of a Request

Comment (quancheng-zhao) [reply src/api-explorer/v4-0/ConcurRequest.swagger2.json]: Done

File: src/api-explorer/v4-0/ConcurRequest.swagger2.json (hunk 1 above)
Comment (quancheng-zhao) [reply src/api-explorer/v4-0/ConcurRequest.swagger2.json]: Done

**Review by quancheng-zhao — APPROVED** [review]

**Comment by RoAnnC1** [comment]
@ArjitGarg-SAP Are you ready for the doc team to review? If you're still making changes just let us know when you're done.

**Comment by ArjitGarg-SAP** [comment]
@RoAnnC1 yes Please you can review the PR, with this PR as well - https://github.com/SAP-docs/preview.developer.concur.com/pull/797

**Review by RoAnnC1 — APPROVED** [review]
:shipit:

**Pull request merged**